roles/mysql/tasks/main.yml

```diff
@@ -0,0 +1,199 @@
+---
+# tasks file for mysql
+#
+- name: MySQL | configure apt repositories
+  ansible.builtin.apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: "0xF1656F24C74CD1D8"
+
+- name: MySQL | add apt repositories
+  ansible.builtin.apt_repository:
+    repo: deb [arch=amd64] https://mirrors.gigenet.com/mariadb/repo/10.2/ubuntu {{ ansible_distribution_release }} main
+    state: present
+    update_cache: true
+
+- name: MySQL | Install database client packages
+  apt:
+    name: "{{ mysql_client_packages }}"
+    state: present
+    install_recommends: false
+    update_cache: true
+
+- name: MySQL | Install python database client packages
+  pip:
+    name: mysqlclient
+  when:
+    - running_on_server
+
+- name: MySQL | install mysql
```

Comment:
I'd add `server` to the task name and to the variable here for clarity ("install mysql server" and `mysql_server_packages`).